src/array.jl

```diff
@@ -364,15 +364,16 @@ arraytype(A::CatArray...) = NullableCategoricalArray
 
 function vcat(A::CatArray...)
     newlevels, ordered = mergelevels(map(levels, A)...)
+    ordered &= any(isordered, A)
```

Comment:
This PR does so that adding an array without any levels wouldn't conflict with any existing ordering. But vcat of arrays with only nulls (i.e no levels anywhere) would then be considered ordered even if they all had isordered==false, that was not the intent. This row covers that edge case.